R/SIRD_plot.R

```diff
@@ -1,121 +1,158 @@
-sird_plot <- function() {
-  expr <- quote({
-    sird_equations <- function(time, variables, parameters) {
-      S <- variables[1]
-      I <- variables[2]
-      R <- variables[3]
-      D <- variables[4]
-      N <- variables[5]
-      q <- variables[6]
-      dS <- (input$muBirth * N) - (input$muDeath * S) - (input$betaSIRD * ((S * I) / (N^q)))
-      dI <- (input$betaSIRD * ((S * I) / (N^q))) - (input$gammaSIRD * I) - (input$deltaSIRD * I) - (input$muDeath * I)
-      dR <- (input$gammaSIRD * I) - (input$muDeath * R)
-      dD <- (input$deltaSIRD * I)
-      dN <- dS + dI + dR
-      list(c(dS, dI, dR, dD, dN, q))
-    }
+library(deSolve)
+library(ggplot2)
```

Comment:
Just making sure I understand. This would mean everywhere I use some function from a package, for example 'deSolve' would go from: 
`lsoda(c(...
` 
to 
`
deSolve::lsoda(c(...
`

Reply:
Yes, exactly! 😃